configs/n2sn.yml

```
@@ -6,3 +6,15 @@ pkg_version: "0.3.6"
 extra_packages: ""
 channels: "-c nsls2forge -c defaults"
 docker_upload: "ghcr"
+zenodo_metadata:
+    metadata:
+        title: "SULI Summer"
+        upload_type: "software"
+        description: "This is a SULI summer intern project."
```

Is there a field that we can use for URLs relevant to the packages we upload?
```suggestion
        description: "User tools for the NSLS-II Science Network. https://github.com/NSLS-II/N2SNUserTools"
```